packages/api/package.json

```diff
@@ -13,12 +17,14 @@
     "@babel/runtime-corejs3": "7.16.5",
     "@prisma/client": "3.6.0",
     "crypto-js": "4.1.1",
+    "fast-json-parse": "1.0.3",
     "jsonwebtoken": "8.5.1",
     "jwks-rsa": "2.0.5",
     "md5": "2.3.0",
     "pascalcase": "1.0.0",
-    "pino": "7.3.0",
-    "pino-pretty": "7.3.0",
+    "pino": "7.6.0",
+    "pretty-bytes": "5.6.0",
```

Comment:
So i'm not super sure about having rw-log-formatter in this package, because its only for dev-time correct?

Perhaps it would be better to have this as part of rwjs/internal or rwjs/cli? What do you think?

My concern is that we're adding extra dependencies that'll get shipped to prod

Reply:
I did think about that, but I wanted the formatter to be associated with the logger.

Also, there's no reason if you have full serverful control of your log output that you could not pipe using this bin util -- so, having as part of api seemed better than internal.

Reply:
Though maybe I move the pretty-bytes and other as dev dependencies?